sunpy/coordinates/utils.py

```diff
@@ -265,3 +265,74 @@ def coordinates(self, points=None):
                         obstime=self.obstime,
                         observer=self.observer,
                         frame=Heliocentric).transform_to(self.start_frame)
+
+
+def get_rectangle_coordinates(bottom_left, *, top_right = None, width: u.deg = None, height: u.deg = None):
+    """
+    Specify a rectangular region of interest in spherical world coordinates.
+
+    Parameters
+    ----------
+    bottom_left : `~astropy.coordinates.BaseCoordinateFrame` or `~astropy.coordinates.SkyCoord`
+        The bottom_left coordinate of the rectangle. If a `SkyCoord` it can
+        have shape ``(2,)`` and also define ``top_right``.
```

Comment:
Why "If a SkyCoord..."?  Does the function not work if `bottom_left` is a frame (not a SkyCoord) and has shape `(2,)`?

Reply:
Also, this sentence doesn't make sense.  Having `shape (2,)` and defining `top_right` are two exclusionary circumstances, so shouldn't be joined by "and also".

Reply:
yeah we don't need the "if a SkyCoord" specifier here, this argument can only ever be a coordinate. I think the also is to indicate that this argument can also specify the top right coordinate of the rectangle (not the argument to the function), so clearly needs rewording.